cmd/crc/cmd/start.go

```diff
@@ -321,3 +333,22 @@ func checkDaemonStarted() error {
 	}
 	return daemonclient.CheckIfOlderVersion(v)
 }
+
+func portFallbackWarning() string {
+	var fallbackPortWarning string
+	fallbackPortWarningTmpl := `
+Note: Port %d is Assigned.
```

Comment:
Why is the method called 'Warning' but the string is `Note:`? :)

Reply:
@cfergeau I put the `Warning` instead `Note`.